SWIPs/swip-pss-capability.md

```diff
@@ -0,0 +1,112 @@
+---
+SWIP: <to be assigned>
+title: PSS capabilities
+author: Louis Holbrook <dev@holbrook.no> (https://holbrook.no)
+status: Draft
+type: Core
+created: 2019-09-21
+---
+
+## Abstract
+
+Description of which capability settings that apply in PSS, and how they affect the module's behavior.
+
+
+## Motivation
+
+The capability settings of PSS need to be documented.
+
+To meet use cases where PSS even can be used as the sole active component of Swarm (even without chunk data), it is important that the operation of PSS is as adaptive and flexible as possible.
+
+
+## Specification
+
+The PSS `CapabilityID` is set to `1` by convention. **No implementations of Swarm should use this `CapabilityID` for any other purpose.**
```

Comment:
Should this be documented in whatever spec captures capabilities? As in a RESERVED list